polyglotdb/corpus/summarized.py

```diff
@@ -273,69 +273,40 @@ def baseline_duration(self, annotation, speaker = None):
         if 'buckeye' in self.corpus_name:
             buckeye = True
         
-
+        index = 'label'
         word = getattr(self, self.word_name)
         phone = getattr(self,self.phone_name)
-        if annotation == 'word':
-            q = self.query_graph(word)
-            index = 'label'
-        elif annotation == 'utterance':
-            q = self.query_graph(self.utterance)
+        # if annotation == 'word':
+        #     annotation = word
+        if annotation == 'utterance':
             ## TODO: find a good key for utterances (labels too long anyway and are None)
             index = 'id'
-        elif annotation == 'syllable':
-            q = self.query_graph(self.syllable)
-            index = 'label'
-        else:
-            raise(AttributeError('Annotation type \'{}\' not found.'.format(annotation)))
-            q = None
-            index = None
+            if not self.hierarchy.has_type_property('utterance','label'):
+                raise(AttributeError('Annotation type \'{}\' not found.'.format(annotation)))
+        if annotation == 'syllable':
+            if not self.hierarchy.has_type_property('syllable','label'):
+                raise(AttributeError('Annotation type \'{}\' not found.'.format(annotation)))
+
+        statement = '''
+MATCH (m:phone:{corpus_name}) 
+with m.{index} as l, avg(m.end-m.begin) as dur 
```

Comment:
I can't tell if that's a 1 (one) or and l (L), could you use a different variable as the alias?